NanoAnalysis/test/prod/samplesNano_2024_Data.csv

```diff
@@ -0,0 +1,46 @@
+identifier,process,crossSection=-1,BR=1,execute=True,dataset,prefix,pattern,splitLevel,::variables,::pyFragments,comment
+###,,,,,,,,,,,info taken from: https://docs.google.com/presentation/d/1EHxQcWzw8IxPgCn8hm1prwSP-EktFtiuaEzH8WkQNVY/edit?slide=id.g34e821b3a62_2_0#slide=id.g34e821b3a62_2_0 
+Muon02024Cv1,,-1,,,/Muon0/Run2024C-MINIv6NANOv15-v1/NANOAOD,dbs,root://cms-xrd-global.cern.ch/,2,DATA_TAG=;PD=Muon;PROCESS_CR=True;PROCESS_ZL=True;FILTER_EVENTS=Z;NANOVERSION=15;LEPTON_SETUP=2024;APPLYELECORR=True;APPLYMUCORR=True;APPLYJETCORR=True,,
```

Comment:
as DATA_TAG should not be set, please remove the string `"DATA_TAG=;"` from all lines.